src/beautifiers/prettydiff.coffee

```diff
@@ -117,6 +117,7 @@ module.exports = class PrettyDiff extends Beautifier
         source: text
         lang: lang
         mode: "beautify"
+        crlf: getDefaultLineEnding() # solved issue 1457
```

Comment:
- [ ] Update options documentation to reflect the `eol` option support in Pretty Diff